### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by JonasKruckenberg**
This
- updates license files and package/crate manifest license statements
- adds lincese headears + checks
- adds rustfmt config
- updates rust edition
- updates tauri version
- brings over the package.json export section from `tauri-plugin-log`

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,11 +3,12 @@ name = "tauri-plugin-sql"
...
 authors = ["Tauri Programme within The Commons Conservancy"]
 description = "A Tauri plugin for easily interfacing with SQL databases"
-edition = "2018"
+license = "Apache-2.0 OR MIT"
+edition = "2021"
 exclude = ["/examples", "/webview-dist", "/webview-src", "node_modules"]
 
 [dependencies]
-tauri = "1.0.0-beta.8"
+tauri = { git = "https://github.com/tauri-apps/tauri", branch = "next" }
```
Comment (FabianLars) [inline Cargo.toml]: Can we keep this on beta.8 for now? This plugin is pretty much the most used one and idk if breaking this is a good idea right now 🤔 
Same for the examples then ofc

Comment (JonasKruckenberg) [reply Cargo.toml]: Fine by me too, my intention was to get all plugins to a similar state and that includes using the latest tauri version.
Buuuut I guess you're right, we've told people to use this plugin already and they might not be happy with that (event though it's technically 0.x)

Comment (JonasKruckenberg) [reply Cargo.toml]: reverted!

**Comment by FabianLars** [comment]
There are still one or two problems with `yarn build` here.
1) the syntax in build:esm is wrong and just throws an error (`--format esm,cjs` instead should work, but unecessary if we drop cjs)
2) with type:module, tsup outputs the esm as a .js file, which is fine imo anyway since it seems we want to switch to esm only anyway. kinda surprised by this because iirc we came to a different conclusion a few weeks ago. But the less commonjs, the better if you ask me lolol
3) we don't output a minified file, so `"browser": "webview-dist/index.min.js",` should cause problems too.

**Comment by JonasKruckenberg** [comment]
> 1. the syntax in build:esm is wrong and just throws an error (`--format esm,cjs` instead should work, but unecessary if we drop cjs)

Didn't touch that 🤷🏻‍♂️ but I can fix it sure!

> 2. with type:module, tsup outputs the esm as a .js file, which is fine imo anyway since it seems we want to switch to esm only anyway. kinda surprised by this because iirc we came to a different conclusion a few weeks ago. But the less commonjs, the better if you ask me lolol
> 3. we don't output a minified file, so `"browser": "webview-dist/index.min.js",` should cause problems too.

I left a comment under the commit that introduced `tsup` stating exactly this, but I don't really know how to move forwards here. The reason why we had a minified browser build (and why we're checking the dist folder into git at all) is so that people who don't want to setup a full build system for trying out tauri can just use a script tag. *Or* - something that denjell apparently likes - use this plugin by making it a git submodule of ones app.
And as all other plugins (even `@tauri-apps/api`) export a minified file, I'd like this plugin to do the same.

But 👍🏻 from me on removing cjs (I think building cjs was a regression bc I had removed it a long time ago)

**Review by FabianLars — APPROVED** [review]

**Pull request merged**